blob/blob.go

```diff
@@ -138,3 +139,37 @@ func (b *Blob) UnmarshalJSON(data []byte) error {
 	b.namespace = blob.Namespace
 	return nil
 }
+
+// createBlobs creates and returns blobs from the passed shares. In case if blob is incomplete, then
+// the shares of the incomplete blob will be returned.
+func createBlobs(appShares []shares.Share) ([]*Blob, []shares.Share, error) {
+	blobs := make([]*Blob, 0)
+	for {
+		if len(appShares) == 0 {
+			return blobs, nil, nil
+		}
+
+		length, err := appShares[0].SequenceLen()
+		if err != nil {
+			return nil, nil, err
+		}
+
+		amount := shares.SparseSharesNeeded(length)
+		if amount > len(appShares) {
+			return blobs, appShares, nil
+		}
```

Comment:
can you document this case in a bit more detail? When would it happen that the shares would not be able to form a complete blob for a namespace at this point? Does this mean the caller wasn't able to recover the full shares of the namespace?

Reply:
changed the comment of the whole method